packages/aic/src/types.ts

```diff
@@ -0,0 +1,41 @@
+export type ErrorMessage = {
+    error: string
+    cause?: ErrorMessage
+    [key: string]: Value
+  }
+  
+export type Value =
+    | null
+    | boolean
+    | number
+    | string
+    | Value[]
+    | { [key: string]: Value }
+    | undefined
+
+export type TidString = string
+export type Signature = string
+export type DidKeyString = string
+export type Patch = ['put' | 'del', (string | number)[], Value]
+export type Diff = {
```

Comment:
Does a diff need to include _its_ TID?

Cause ticks don't stick around, right?

Reply:
The diff dose not know its own tid at the time it is signed.
The tid is assigned by the consortium when it puts the diff in the order.